bundles/binding/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/model/ADZone.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.alarmdecoder.internal.model;
+
+/**
+ * A class to hold our zone state
+ *
+ * @author Sean Mathews <coder@f34r.com>
+ * @since 1.6.0
+ */
+
+public class ADZone {
+
+    private int _id = 0;
+    private double _ts = 0;
```

Comment:
Certainly remove it for now if you're not using it, and if you need it later, consider instead using `System.currentTimeMillis()` instead.